core/src/test/scala/io/delta/tables/DeltaTableSuite.scala

```diff
@@ -160,6 +160,14 @@ class DeltaTableSuite extends QueryTest
     }
   }
 
+  test("isDeltaTable - with table name") {
```

Comment:
add test for when defined by both name and path such that exists=true. add test for when only defined by path, but in current directory and provide rel path (counter example provided above)

Reply:
> add test for when defined by both name and path such that exists=true

This has been added

Reply:
> add test for when only defined by path, but in current directory and provide rel path

I am seeing that save("path") - requires an absolute path and using option("path", "relpath") is placing the managed data under /hive/warehouse. Not relative to curr dir.